include/splinterdb/splinterdb.h

```diff
@@ -136,7 +162,7 @@ typedef struct splinterdb splinterdb;
 // But cfg->data_cfg will be referenced by the returned splinterdb object
 // So it must live at least as long as the splinterdb
 int
-splinterdb_create(const splinterdb_config *cfg, splinterdb **kvs);
+splinterdb_create(splinterdb_config *cfg, splinterdb **kvs);
```

Comment:
See the definition for the reasons why changed from `const` to non-const. Same applies to `splinterdb_open()` below.

Reply:
Has been fixed.